src/App.tsx

```diff
@@ -2,12 +2,14 @@ import { Route, Routes } from "react-router-dom";
 
 import Layout from "./components/layout/Layout";
 import Home from "./pages/home/Home";
+import DetailRecepie from "./pages/home/recepie";
 
 export default function App() {
   return (
     <Layout>
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/recepie" element={<DetailRecepie />} />
```

Comment:
I think a small typo slipped in and persisted through out the changes ,`recepie` should be `recipe`

Reply:
yeah ... i am really sorry for the typo.  Though i added new route with params .  and i am using some dummy api to mimic the "fetching by id" functionality . you guys should check it once more by changing the url by "/recipe/id" where id can be 1,2,3,4 etc

Reply:
You can add the code spell checker extension on your visual studio code.
![image](https://github.com/chingu-voyages/v46-tier2-team-14/assets/8060579/44101645-c066-471a-8e98-83b4d004fd15)

Reply:
oh... thanks.... btw that's really great